FFXIVClientStructs/FFXIV/Client/Network/NetworkModule.cs

```diff
@@ -0,0 +1,49 @@
+using System.ComponentModel.DataAnnotations;
+using FFXIVClientStructs.FFXIV.Application.Network;
+using FFXIVClientStructs.FFXIV.Client.System.String;
+using Unknown = nint;
+
+namespace FFXIVClientStructs.FFXIV.Client.Network;
+
+[StructLayout(LayoutKind.Explicit, Size = 0xB60)]
+public unsafe partial struct NetworkModule {
+    [FieldOffset(0x028)] public int LobbyCount;
+    [FixedSizeArray<uint>(0xC)] [FieldOffset(0x02C)]
+    public fixed byte LobbyPorts[0xC * 0x4];
+    [FixedSizeArray<Utf8String>(0xC)] [FieldOffset(0x060)]
+    public fixed byte LobbyHosts[0xC * 0x68];
```

Comment:
Using `FixedSizeArray` on LobbyPorts is not needed, because you can use `fixed uint`.
```suggestion
    [FieldOffset(0x02C)] public fixed uint LobbyPorts[12];
    [FixedSizeArray<Utf8String>(12)]
    [FieldOffset(0x060)] public fixed byte LobbyHosts[12 * 0x68];
```